fractals/sierpinski_triangle.py

```diff
@@ -82,3 +82,4 @@ def triangle(
 
     vertices = [(-175, -125), (0, 175), (175, -125)]  # vertices of triangle
     triangle(vertices[0], vertices[1], vertices[2], int(sys.argv[1]))
+    turtle.Screen().exitonclick()
```

Comment:
```suggestion
    turtle.Screen().exitonclick()

```
Make sure the file ends in an empty line